### PR TITLE
Oxidation replace protocol organization

### PR DESCRIPTION
**Pull request by TimeEngineer**

### CODE REVIEW

**Comment by TimeEngineer** [comment]
Missing test with sequester

File: src/protocol/organization.rs [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+                realms,
+                users,
+                active_users,
+                users_per_profile_detail,
+            }),
+        ))
+    }
+
+    #[getter]
+    fn data_size(_self: PyRef<'_, Self>) -> PyResult<u64> {
```
Comment (FirelightFlagboy) [inline src/protocol/organization.rs:135]:
```suggestion
    fn data_size(self: PyRef<'_, Self>) -> PyResult<u64> {
```

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: src/protocol/organization.rs (hunk 1 above)
Comment (TimeEngineer) [reply src/protocol/organization.rs:135]: self is keyword

File: src/protocol/organization.rs [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+        Ok(match _self.as_ref().0 {
+            organization_stats::Rep::Ok { data_size, .. } => data_size,
+            _ => return Err(PyNotImplementedError::new_err("")),
+        })
+    }
+
+    #[getter]
+    fn metadata_size(_self: PyRef<'_, Self>) -> PyResult<u64> {
+        Ok(match _self.as_ref().0 {
```
Comment (FirelightFlagboy) [inline src/protocol/organization.rs:144]:
```suggestion
    fn metadata_size(self: PyRef<'_, Self>) -> PyResult<u64> {
        Ok(match self.as_ref().0 {
```

File: src/protocol/organization.rs (hunk 1 above)
Comment (FirelightFlagboy) [reply src/protocol/organization.rs:135]: you can remove the `_`

Comment (FirelightFlagboy) [reply src/protocol/organization.rs:135]: I want something like `Pin` https://doc.rust-lang.org/std/pin/index.html#pinning-is-not-structural-for-field

Comment (TimeEngineer) [reply src/protocol/organization.rs:135]:
```shell
error[E0658]: `pyo3::PyRef<'_, organization::OrganizationStatsRepOk>` cannot be used as the type of `self` without the `arbitrary_self_types` feature                                                   
   --> src/protocol/organization.rs:135:24
    |
135 |     fn data_size(self: PyRef<'_, Self>) -> PyResult<u64> {
    |                        ^^^^^^^^^^^^^^^
    |
    = note: see issue #44874 <https://github.com/rust-lang/rust/issues/44874> for more information
    = help: consider changing to `self`, `&self`, `&mut self`, `self: Box<Self>`, `self: Rc<Self>`, `self: Arc<Self>`, or `self: Pin<P>` (where P is one of the previous types except `Self`)
```

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**